torch/nn/parameter.pyi

```diff
@@ -1,4 +1,7 @@
 from .. import Tensor
+import builtins
 
 class Parameter(Tensor):
+    def __init__(self, data: Tensor, requires_grad: builtins.bool): ...
```

Comment:
We support Python 2 as well, so we can't use python3-style type annotations. Please switch to the Python 2 style annotations.

Reply:
Oh sorry, I didn't realize this was in a .pyi file. Please ignore this comment.